s4e-web/src/app/views/settings/components/generic-list-view/generic-list-view.component.html

```diff
@@ -0,0 +1,52 @@
+<header class="page__header">
+  <h1 i18n>
+    <ng-content select="[caption]"></ng-content>
+  </h1>
+  <p i18n>
+    <ng-content select="[description]"></ng-content>
+  </p>
+</header>
+<section class="page__content">
+  <div>
+    <input data-test-generic-list-query [formControl]="queryFc" placeholder="Wyszukaj..." i18n-placeholder="">
+  </div>
+
+  <ng-container *ngIf="error == null; else errorView">
```

Comment:
What will happen if error will be `undefined` or `NaN`? Could you change `error == null` to `!!error`?

Reply:
`error == null` covers both `undefined` as well as `null`. I believe that if error is anything else that means we have some serious problems in the application and we should not try to accomodate it in the template, but rather catch it much earlier.

`!!error` will also convert `0`, `''` etc.

Reply:
ok